oml/models/resnet.py

```diff
@@ -152,6 +160,26 @@ def draw_gradcam(self, image: np.ndarray) -> np.ndarray:
         img_with_grads = show_cam_on_image(image / 255, gray_image)
         return img_with_grads
 
+    @classmethod
+    def from_pretrained(cls, weights: str) -> "ResnetExtractor":
+        if weights.lower().endswith("_default") or weights.lower().endswith("_imagenet_v1"):
```

Comment:
Why do we need some custom code here instead of a declaration in `pretrained_models`?

In the case of ViT we have a lot of external checkpoints and they are in `pretrained_models`. But for some reason, it's not the case for resnet torchvision checkpoints.

Reply:
If you remove custom code, it will be possible to move `from_pretrained` to `Extractor`, read `pretrained_models`, remove keys with `remove_unused_kwargs` and pass them to `__init__`

Reply:
It's a mess with ResNet, I know. I already created an issue to fight with that in the next PR. It will be easier to review then. After that, I can unify `from_pretrained` if possible (or, I will do it altogether with refactoring of `ResnetExtractor`)